js/classes/entities/foreground/hud.js

```diff
@@ -135,10 +167,18 @@ class Hud {
     update() {
         // if the user clicks on the pause/play button, toggle GAME.running
         if (GAME.user.firing && this.pauseButton.isMouseOver()) {
-            GAME.running = !GAME.running;
-            ASSET_MGR.playAudio(SFX.UI_HIGH_BEEP.path, SFX.UI_HIGH_BEEP.volume);
-            // TODO: open options menu here
-
+            ASSET_MGR.playSFX(SFX.UI_HIGH_BEEP.path, SFX.UI_HIGH_BEEP.volume);
+            if (GAME.running) {
+                GAME.running = false;
```

Comment:
Why are we setting the GAME.running to false when the condition checks if it is true? Can you explain this?